### ISSUE TITLE
Operation timed out

### ISSUE THREAD

**Issue by ilinuxadmin** [oerdnj/deb.sury.org#895]
Few of my servers cannot download updates.

```
# apt update
Hit:1 http://security.debian.org/debian-security stretch/updates InRelease
Ign:2 http://ftp.uk.debian.org/debian stretch InRelease
Get:3 http://ftp.uk.debian.org/debian stretch-updates InRelease [91.0 kB]
Hit:4 http://ftp.uk.debian.org/debian stretch Release
Err:6 https://packages.sury.org/php stretch InRelease                                                                                               
  Operation timed out after 120000 milliseconds with 0 out of 0 bytes received
Fetched 91.0 kB in 2min 0s (758 B/s)
Reading package lists... Done
Building dependency tree       
Reading state information... Done
All packages are up to date.
W: Failed to fetch https://packages.sury.org/php/dists/stretch/InRelease  Operation timed out after 120000 milliseconds with 0 out of 0 bytes received
W: Some index files failed to download. They have been ignored, or old ones used instead.
```

Debian 9 x64

**Comment by ilinuxadmin** [comment]
After contacting the datacenter it appeared to be an issue with the Internet connectivity.

**Comment by oerdnj** [comment]
That would be my first guess anyway...

**Comment by ilinuxadmin** [comment]
Actually it was but then after running some tests we found out that only few IP addresses are unreachable so, we suspected that maybe the issue was not at our end. 
After contacting the datacenter they let us know that they are having some troubles and they are working on it.
We just found out that there is a restriction on those servers' IP addresses because of their location and they cannot download packages from packages.sury.org so, we had to use proxy to fix the issue temporarily.
